addons/point_of_sale/tests/test_frontend.py

```diff
@@ -57,6 +57,7 @@ def setUpClass(cls, chart_template_ref=None):
             'password': 'pos_admin',
             'groups_id': [
                 (4, cls.env.ref('point_of_sale.group_pos_manager').id),
+                (4, cls.env.ref('account.group_account_invoice').id),
```

Comment:
This shouldn't be necessary. Something could be wrong if you'll need to add this.